--- lang/php/Portfile.orig	2020-01-27 10:49:02.000000000 +0100
+++ lang/php/Portfile	2020-01-27 10:49:02.000000000 +0100
@@ -298,6 +298,11 @@
         configure.args-append ac_cv_decimal_fp_supported=no
     }
     
+    if {[vercmp ${branch} 7.2] >= 0} {
+        configure.args-append --with-password-argon2=${prefix}
+        depends_lib-append  port:argon2
+    }
+
     configure.universal_args-delete --disable-dependency-tracking
     
     test.run            yes
